ui/client/components/graph/node-modal/editors/Validators.ts

```diff
@@ -93,12 +95,30 @@ export const regExpValueValidator = (pattern: string, message: string, descripti
 export const literalIntegerValueValidator: Validator = {
   //Blank value should be not validate - we want to chain validators
   isValid: value => isEmpty(value) || literalRegExpPattern("^-?[0-9]+$").test(value),
-  message: () => i18next.t("literalIntegerValueValidator.message",  "This field value has to be an integer number"),
-  description: () => i18next.t("literalIntegerValueValidator,description", "Please fill field by proper integer type"),
+  message: () => i18next.t("literalIntegerValueValidator.message", "This field value has to be an integer number"),
+  description: () => i18next.t("literalIntegerValueValidator.description", "Please fill field by proper integer type"),
   handledErrorType: HandledErrorType.InvalidIntegerLiteralParameter,
   validatorType: ValidatorType.Frontend,
 }
 
+export const minimalNumberValidator = (minimalNumber: number): Validator => ({
+  //Blank value should be not validate - we want to chain validators
+  isValid: value => isEmpty(value) || Number(value) >= minimalNumber,
+  message: () => i18next.t("minimalNumberValidator.message", `This field value has to be a number greater than or equal to ${minimalNumber}`),
+  description: () => i18next.t("minimalNumberValidator.description", "Please fill field by proper number"),
+  handledErrorType: HandledErrorType.InvalidPropertyFixedValue,
```

Comment:
You use handledErrorType in two cases: minimalNumberValidator and maximalNumberValidator. I think there can be a some problem with matching these validators.. You can write FE tests or test it  manually at FE.